### ISSUE TITLE
The trained weights do not reproduce the results.

### ISSUE THREAD

**Issue by yy13138** [Haozhe-Xing/SARNet#6]
I'm trying to train on CAMO with 1000 images, hoping to get matching test results, but on the test set (250 images), my Smeasure is always around 40, which is 40% lower and faster than the one in the paper. Is it my training method?

**Comment by Haozhe-Xing** [comment]
the dataset you are using maybe not right,  the train datasets contains CAMO and COD10K dataset, the number of the whole train dataset is about 4000.

**Comment by yy13138** [comment]
Thank you. I'll reorganize right away. You're so kind and timely.

**Comment by Haozhe-Xing** [comment]
you are welcomed, thanks for your attention on our project.

**Comment by yy13138** [comment]
COD10K-v3 dataset The training set has 6000 images, are you using this? What about other versions? Thank you for your reply. Thank you.

**Comment by yy13138** [comment]
COD10K-v3 dataset The training set has 6000 images, are you using this? What about other versions? Thank you for your reply. Thank you.

**Comment by Haozhe-Xing** [comment]
I used the previous version, thanks for the reminder, I will update the dataset to git when I have time.

**Comment by Haozhe-Xing** [comment]
you can use the new version to train your model

**Comment by yy13138** [comment]
First of all, thank you for your kind reply. I have found the required data set according to your tips. Thank you again. Here is the dataset address, I think anyone who needs data will also follow the URL to find the correct data, wish you good luck in your research.

https://drive.google.com/file/d/1D9bf1KeeCJsxxri6d2qAC7z6O1X_fxpt/view
